### ISSUE TITLE
Cityperson json label to yolo label

### ISSUE THREAD

**Issue by swaingotnochill** [ultralytics/yolov5#2727]
## ❔Question
I am trying to evaluate the model(  pretrained weights) on Cityperson dataset. I wish to convert the labels into yolo format, but can't do it. Do you have any idea how should I approach this? Also, is there any other thing I should take care for the evaluation?

## Additional context
I already have the labels for the cityperson dataset. The question is I wish to replace all the pedestrian, rider, etc with a single class and also am not sure of the object class Id.
Any suggestions is valuable.

**Comment by glenn-jocher** [comment]
@swaingotnochill you should export your labels into YOLO format and create a citypersons.yaml file for this dataset. Then you can train and test normally using train.py and test.py. See Train Custom Data tutorial to get started:

## YOLOv5 Tutorials

* [Train Custom Data](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data)&nbsp; 🚀 RECOMMENDED
* [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/)&nbsp; ☘️ RECOMMENDED
* [Weights & Biases Logging](https://github.com/ultralytics/yolov5/issues/1289)&nbsp; 🌟 NEW
* [Supervisely Ecosystem](https://docs.ultralytics.com/yolov5/tutorials/model_export8)&nbsp; 🌟 NEW
* [Multi-GPU Training](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training)
* [PyTorch Hub](https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading)&nbsp; ⭐ NEW
* [ONNX and TorchScript Export](https://docs.ultralytics.com/yolov5/tutorials/model_export)
* [Test-Time Augmentation (TTA)](https://docs.ultralytics.com/yolov5/tutorials/test_time_augmentation)
* [Model Ensembling](https://docs.ultralytics.com/yolov5/tutorials/model_ensembling)
* [Model Pruning/Sparsity](https://docs.ultralytics.com/yolov5/tutorials/model_pruning_and_sparsity)
* [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution)
* [Transfer Learning with Frozen Layers](https://docs.ultralytics.com/yolov5/tutorials/transfer_learning_with_frozen_layers)&nbsp; ⭐ NEW
* [TensorRT Deployment](https://github.com/wang-xinyu/tensorrtx)

**Comment by NguyenThanhBinh98** [comment]
> ## Question
> I am trying to evaluate the model( pretrained weights) on Cityperson dataset. I wish to convert the labels into yolo format, but can't do it. Do you have any idea how should I approach this? Also, is there any other thing I should take care for the evaluation?
> 
> ## Additional context
> I already have the labels for the cityperson dataset. The question is I wish to replace all the pedestrian, rider, etc with a single class and also am not sure of the object class Id. Any suggestions is valuable.

**Comment by NguyenThanhBinh98** [comment]
Hi swaingotnochill . Can you share with me the labeled  cityperson dataset . I want train Yolov5 with that

**Comment by glenn-jocher** [comment]
@NguyenThanhBinh98 @swaingotnochill yes it would be great to create a Cityperson.yaml for everyone to be able to autodownload this dataset. 

Please see our ✅ [Contributing Guide](https://docs.ultralytics.com/help/contributing/) to get started.

**Comment by swaingotnochill** [comment]
@NguyenThanhBinh98 @glenn-jocher  I don't have the dataset as of now. But the dataset is available in https://www.cityscapes-dataset.com/ . I have wrote the script for converting the cityperson labels to yolo lables. If that's ok, maybe I can create an PR for that.

**Comment by glenn-jocher** [comment]
@swaingotnochill yes great! If you can create a Cityscapes.yaml similar to our other dataset yamls and start populating it what you have, that will help get this started.

**Comment by swaingotnochill** [comment]
@glenn-jocher I don't think that is possible. I had a conversation with a CityScape person, and they don't allow free download of the datasets. Their dataset is available only for research/education purpose. So , they don't have an Open Download Link.  When I downloaded the dataset, I had to submit my University Student details to be verified. This is a troublesome process, but that is how they operate :(

**Comment by glenn-jocher** [comment]
@swaingotnochill yes this should be ok. We have the same setup in place with the xView dataset. The users download xView zips manually to /datasets, and then run `python train.py --data xView.yaml`, and we automatically extract and convert to YOLO format and start training.

Also we are unaffected by dataset licensing since we are not an end user. It would be up to end users (like yourself) to download CityScape zips first and ensure they are following license guidelines.

https://github.com/ultralytics/yolov5/blob/99a45bad81535728be67c9871fd5513588d8d471/data/xView.yaml#L1-L9

**Comment by codename-cn** [comment]
@swaingotnochill you don't necessarily have to provide the dataset itself. However, you could have provided the script for the conversion.

**Comment by glenn-jocher** [comment]
@codename-cn Sharing your script for converting Cityperson labels to YOLO format would indeed be very helpful for the community! If you could create a pull request with your script, that would be fantastic. This way, others can benefit from your work once they have access to the dataset themselves. 😊👍
